lib/pdf/pdf.go

```diff
@@ -117,17 +119,38 @@ func (g *GoFPDF) AddPDFPage(png []byte, boardPath []string, themeID int64, fill
 	g.pdf.CellFormat(pageWidth-prefixWidth-headerMargin, headerHeight, boardName, "", 0, "", false, 0, "")
 
 	// Draw image
-	g.pdf.ImageOptions(strings.Join(boardPath, "/"), (pageWidth-imageWidth)/2, headerHeight+(pageHeight-imageHeight)/2, imageWidth, imageHeight, false, opt, 0, "")
+	imageX := (pageWidth - imageWidth) / 2
+	imageY := headerHeight + (pageHeight-imageHeight)/2
+	g.pdf.ImageOptions(strings.Join(boardPath, "/"), imageX, imageY, imageWidth, imageHeight, false, opt, 0, "")
+
+	// Draw external links
+	for _, shape := range shapes {
+		if shape.Link != "" {
+			viewboxX, err := strconv.ParseFloat(viewboxSlice[0], 64)
+			if err != nil {
+				return err
+			}
+			viewboxY, err := strconv.ParseFloat(viewboxSlice[1], 64)
+			if err != nil {
+				return err
+			}
```

Comment:
these are the same for all shapes so this can be outside the loop, or passed in as params instead of the slice